packages/design-system/src/stitches.config.ts

```diff
@@ -257,6 +258,10 @@ export const {
       4: "400",
       max: "999",
     },
+    borderWidths: {
```

Comment:
it probably doesn't make sense to have borderWidths a part of the design system if they are not simplifying anything and basically translate 1:1, please check with @taylornowotny

Reply:
Yeah, makes sense. I don't think we'll ever want to change all borders from `1px` to `2px`. Will revert.